homeworks/08-functional-reactive-programming/src/drawing.ts

```diff
@@ -0,0 +1,32 @@
+import {AnimalImage, BricksImage, ReticleImage, WindowImage} from "./assets"
+import {Entity, State} from "./domain"
+
+export const CellSize = 50
+
+const EntityImages = {
+    [Entity.Bricks]: BricksImage,
+    [Entity.Window]: WindowImage,
+    [Entity.Animal]: AnimalImage,
+}
+
+export function draw(context: CanvasRenderingContext2D, state: State): void {
+    const drawEntity = (entity: Entity, colIdx: number, rowIdx: number): void =>
+        context.drawImage(EntityImages[entity], colIdx * CellSize, rowIdx * CellSize, CellSize, CellSize)
+
+    context.clearRect(0, 0, context.canvas.width, context.canvas.height)
+
+    context.fillStyle = "black"
```

Comment:
Default color is already black

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillStyle

Reply:
I think it's an OK practice to leave it explicit so that the reader of the code doesn't have to look that up in the docs.